src/Raven.Server/Web/System/DatabasesHandler.cs

```diff
@@ -83,6 +86,83 @@ await using (var writer = new AsyncBlittableJsonTextWriter(context, ResponseBody
             }
         }
 
+        [RavenAction("/topology/modify", "POST", AuthorizationStatus.ValidUser, EndpointType.Read)]
+        public async Task ModifyTopology()
+        {
+            var dbName = GetQueryStringValueAndAssertIfSingleAndNotEmpty("name").Trim();
+            var raftRequestId = GetRaftRequestIdFromQuery();
+
+            await ServerStore.EnsureNotPassiveAsync();
+            using (ServerStore.ContextPool.AllocateOperationContext(out TransactionOperationContext context))
+            using (context.OpenReadTransaction())
```

Comment:
let's avoid keep a read tx here.

Reply:
Cant do that because of this line:
https://github.com/ravendb/ravendb/blob/b13fe40dc9e53ae42121aa52333f8d2255bc4fcd/src/Raven.Server/Web/System/DatabasesHandler.cs#L111

Reply:
Done